/home/augusto/miniconda3/envs/sr/lib/python3.8/site-packages/trl/trainer/ppo_trainer.py:239: UserWarning: No dataset is provided. Make sure to set config.batch_size to the correct value before training.
  warnings.warn(
avaliation.py:112: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  query_tensors = list(torch.tensor(query_tensors))
Index(['type', 'name', 'expression'], dtype='object')
self.infix=x**5/5 + x**2/2
self.infix=None
self.infix=4*x**(3/2)/3 + 3*x**2/2
self.infix=None
self.infix=x**2/2 + 4*log(x)
self.infix=x*exp(2*x) + x
<lambdifygenerated-4>:2: RuntimeWarning: overflow encountered in exp
  return x*exp(2*x) + x
<lambdifygenerated-4>:2: RuntimeWarning: overflow encountered in multiply
  return x*exp(2*x) + x
Traceback (most recent call last):
  File "avaliation.py", line 130, in <module>
    exprs, rewards = reward_pipeline(response_tensors, data)
  File "avaliation.py", line 36, in reward_pipeline
    exprs = [Expression(prefix.strip().split(" "), data) for prefix in prefixes]
  File "avaliation.py", line 36, in <listcomp>
    exprs = [Expression(prefix.strip().split(" "), data) for prefix in prefixes]
  File "/home/augusto/symbolic/Expression.py", line 29, in __init__
    self.score = self.compute_reward(data.y, method=method)
  File "/home/augusto/symbolic/Expression.py", line 45, in compute_reward
    reward = self.nrmse(y, self.y_pred)
  File "/home/augusto/symbolic/Expression.py", line 59, in nrmse
    reward = mean_squared_error(y, y_pred, squared=False)
  File "/home/augusto/miniconda3/envs/sr/lib/python3.8/site-packages/sklearn/utils/_param_validation.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/home/augusto/miniconda3/envs/sr/lib/python3.8/site-packages/sklearn/metrics/_regression.py", line 474, in mean_squared_error
    y_type, y_true, y_pred, multioutput = _check_reg_targets(
  File "/home/augusto/miniconda3/envs/sr/lib/python3.8/site-packages/sklearn/metrics/_regression.py", line 101, in _check_reg_targets
    y_pred = check_array(y_pred, ensure_2d=False, dtype=dtype)
  File "/home/augusto/miniconda3/envs/sr/lib/python3.8/site-packages/sklearn/utils/validation.py", line 957, in check_array
    _assert_all_finite(
  File "/home/augusto/miniconda3/envs/sr/lib/python3.8/site-packages/sklearn/utils/validation.py", line 122, in _assert_all_finite
    _assert_all_finite_element_wise(
  File "/home/augusto/miniconda3/envs/sr/lib/python3.8/site-packages/sklearn/utils/validation.py", line 171, in _assert_all_finite_element_wise
    raise ValueError(msg_err)
ValueError: Input contains infinity or a value too large for dtype('float64').